lardataobj/MCBase/MCMiniPart.h

```diff
@@ -0,0 +1,153 @@
+/**
+ * \file MCMiniPart.h
+ *
+ * \ingroup MCBase
+ *
+ * \brief Class def header for MCMiniPart data container
+ *
+ * @author Temigo
+ */
+
+/** \addtogroup MCBase
+
+    @{*/
+#ifndef MCMINIPART_H
+#define MCMINIPART_H
+
+// LArSoft
+//#include "lardataobj/MCBase/MCStep.h"
+#include "lardataobj/MCBase/MCLimits.h" // kINVALID_X
+#include "nusimdata/SimulationBase/MCParticle.h"
+#include "nusimdata/SimulationBase/MCTruth.h" // simb::Origin_t
+
+// ROOT libraries
+#include <TVector3.h>
+
+// STL
+//#include <set>
+#include <utility> // std::pair<>
+#include <vector>
+
+#include "TLorentzVector.h"
+
+namespace sim
+{
+
+  class MCMiniPart {
+
+  public:
+
+    /// Default constructor
+    MCMiniPart() {Reset();}
+
+    /// Default destructor
+    //virtual ~MCMiniPart() = default;
+
+    void Reset() { ResetData(); }
+
+    MCMiniPart(MCMiniPart const &) = default; // copy constructor
+    MCMiniPart& operator=( const MCMiniPart &) = default;
+    MCMiniPart(MCMiniPart&&) = default;
+    MCMiniPart& operator= (MCMiniPart&&) = default;
```

Comment:
Suggest removing user-defined constructors/assignment operators.  The compiler will generate them.

Reply:
Done!